packages/video_player/tizen/src/video_player.cc

```diff
@@ -465,21 +474,23 @@ void VideoPlayer::SendInitialized() {
       }
     }
 
-    // TODO(jsuya):Some streaming resources may have a duration of 0 during the
-    // initialization step. If the duration is 0, it may affect the progress of
-    // video_player and cause unnecessary errors. Therefore, set it to 1
-    // temporarily. In the future, It can be updated depending on
-    // the loading status.
-    if (width != 0 && height != 0 && duration == 0) {
-      duration = 1;
+    // TODO(jsuya):Tizen's player API returns duration as 0 for live
+    // streaming resources. If the duration is 0, it may affect the progress of
+    // video_player and cause unnecessary errors. Therefore, set it to 1.
+    if (uri_.substr(0, 4) == "http" && width != 0 && height != 0 &&
+        duration_ == 0) {
+      duration_ = 1;
+      is_live_ = true;
```

Comment:
Actually, media player has API for streaming info, we can get the is_live info:
```C
  int player_get_adaptive_streaming_info(player_h player, void* adaptive_info,
                                         int adaptive_type);
  bool MediaPlayer::IsLive() {
  int is_live = 0;
  int ret = media_player_proxy_->player_get_adaptive_streaming_info(
      player_, &is_live, PLAYER_ADAPTIVE_INFO_IS_LIVE);
  if (ret != PLAYER_ERROR_NONE) {
    LOG_ERROR("[MediaPlayer] player_get_adaptive_streaming_info failed: %s",
              get_error_message(ret));
    return false;
  }
  return is_live != 0;
}
```
you can refer to :
https://github.com/flutter-tizen/plugins/blob/master/packages/video_player_videohole/tizen/src/media_player_proxy.h

Reply:
Thank you for information. 
However, video_player_tizen should work on common profile(rpi4) and, unless there are special cases, should be able to work on tizen emulator(8.0) . `player_get_adaptive_streaming_info()` has been confirmed to work only on TV devices. I will make a proxy class of mediaplayer and modify the code like videohole. but, the `duration == 0` code can be kept for rpi (common profile).